make build-cross fails on ppc64le platform

[provide a description of the issue]

##### Version
openshift master branch

##### Steps To Reproduce
1. Clone openshift origin master branch
2. make build-cross

##### Current Result
```
[root@localhost origin]# make build-cross
rm -rf _output
hack/build-cross.sh
++ Building go targets for linux/ppc64le: images/pod cmd/dockerregistry cmd/gitserver pkg/sdn/plugin/sdn-cni-plugin vendor/github.com/containernetworking/cni/plugins/ipam/host-local vendor/github.com/containernetworking/cni/plugins/main/loopback examples/hello-openshift examples/deployment
++ Building go targets for linux/amd64: cmd/openshift cmd/oc
# github.com/openshift/origin/pkg/cmd/util/tokencmd
pkg/cmd/util/tokencmd/negotiator_gssapi.go:25: undefined: gssapi.Lib
pkg/cmd/util/tokencmd/negotiator_gssapi.go:32: undefined: gssapi.Name
pkg/cmd/util/tokencmd/negotiator_gssapi.go:34: undefined: gssapi.CtxId
pkg/cmd/util/tokencmd/negotiator_gssapi.go:43: undefined: gssapi.CredId
[ERROR] PID 25650: hack/common.sh:153: `local -a binaries=("$@")` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/common.sh:153: `local -a binaries=("$@")`
[INFO] 		  2: hack/build-cross.sh:81: os::build::build_binaries
[INFO]   Exiting with code 1.
make: *** [build-cross] Error 1
[root@localhost origin]#
```
##### Expected Result

make build-cross should pass

##### Additional Information
[try to run `$ oadm diagnostics` or `oc adm diagnostics` command if possible]
[if you are reporting issue related to builds, provide build logs with `BUILD_LOGLEVEL=5`]
[consider attaching output of the `$ oc get all -o json -n <namespace>` command to the issue]
[visit https://docs.openshift.org/latest/welcome/index.html]

--- a/hack/build-cross.sh
+++ b/hack/build-cross.sh
@@ -8,8 +8,7 @@ host_platform="$(os::build::host_platform)"
 
 # Set build tags for these binaries
 readonly OS_GOFLAGS_TAGS="include_gcs include_oss"
-readonly OS_GOFLAGS_TAGS_LINUX_PPC64LE="gssapi"
-readonly OS_GOFLAGS_TAGS_LINUX_AMD64="gssapi"
+readonly OS_GOFLAGS_TAGS_$(os::build::platform_arch)="gssapi"
 
 # by default, build for these platforms
 platforms=(

--- a/hack/common.sh
+++ b/hack/common.sh
@@ -46,6 +46,18 @@ function os::build::host_platform_friendly() {
 }
 readonly -f os::build::host_platform_friendly
 
+# This converts from platform/arch to PLATFORM_ARCH, host platform will be
+# considered if no parameter passed
+function os::build::platform_arch() {
+  local platform=${1:-}
+  if [[ -z "${platform}" ]]; then
+    platform=$(os::build::host_platform)
+  fi
+
+  echo $(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
+}
+readonly -f os::build::platform_arch
+
 # os::build::setup_env will check that the `go` commands is available in
 # ${PATH}. If not running on Travis, it will also check that the Go version is
 # good enough for the Kubernetes build.
@@ -216,8 +228,8 @@ os::build::internal::build_binaries() {
         unset GOBIN
       fi
 
-      local platform_gotags_envvar=OS_GOFLAGS_TAGS_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
-      local platform_gotags_test_envvar=OS_GOFLAGS_TAGS_TEST_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')
+      local platform_gotags_envvar=OS_GOFLAGS_TAGS_$(os::build::platform_arch ${platform})
+      local platform_gotags_test_envvar=OS_GOFLAGS_TAGS_TEST_$(os::build::platform_arch ${platform})
 
       if [[ ${#nonstatics[@]} -gt 0 ]]; then
         GOOS=${platform%/*} GOARCH=${platform##*/} go install \

